pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -118,7 +127,7 @@ func TestCMServerDefaults(t *testing.T) {
 					IncrementTimeoutHostPath: 30,
 				},
 			},
-			ContentType:  "",
+			ContentType:  "application/vnd.kubernetes.protobuf",
```

Comment:
does this break any clients that expect to speak to origin types (which we don't have protobuf for yet)?